ledger/participant-integration-api/src/main/scala/platform/apiserver/GrpcServer.scala

```diff
@@ -30,16 +30,16 @@ private[apiserver] object GrpcServer {
   // allow for extra information such as the exception stack trace.
   private val MaximumStatusDescriptionLength = 4 * 1024 // 4 KB
 
-  private def makeBuilder(
+  def owner(
       address: Option[String],
       desiredPort: Port,
       maxInboundMessageSize: Int,
-      sslContext: Option[SslContext],
-      interceptors: List[ServerInterceptor],
+      sslContext: Option[SslContext] = None,
+      interceptors: List[ServerInterceptor] = List.empty,
       metrics: Metrics,
       servicesExecutor: Executor,
       services: Iterable[BindableService],
-  ): NettyServerBuilder = {
+  ): AbstractResourceOwner[ResourceContext, Server] = {
```

Comment:
Isn't this the same as `ResourceOwner[Server]`?

Reply:
Indeed, initially I had some type error there and went for the lazy route of putting there whatever the compiler told me to. 😄 Thanks for spotting this. 🙇🏻

Reply:
Addressed by https://github.com/digital-asset/daml/pull/8660/commits/b466c8ee8a7fca1b80f7ef217f14d8c2dd3e2c80